Sources/TuistCache/Cache/CloudHEADResponse.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import TuistCore
+import TuistSupport
+
+struct CloudHEADResponse: Decodable {
+    public static func existsResource(hash: String, config: Config) throws -> HTTPResource<CloudResponse<CloudHEADResponse>, CloudResponseError> {
+        let url = try URL.apiCacheURL(hash: hash, config: config)
```

Comment:
As I mentioned below, I'd get rid of the `apiCacheURl` and move the code here. I find extensions useful when they encapsulate a well-defined of business logic. In your case, you extended the signature of the method to support `contentMD5`, which is only used by some of the consumers of the method.